preview_generator/preview/builder/image__imconvert.py

```diff
@@ -64,7 +64,7 @@ def __load_mimetypes(cls) -> typing.List[str]:
         mimes = []  # type: typing.List[str]
         for supported in all_supported:
             url = "./FILE.{0}".format(supported)  # Fake a url
-            mime, enc = mimetypes.guess_type(url)
+            mime, enc = mimetypes_storage.guess_type(url)
```

Comment:
`fake_path` instead `url` should be more approriate

Reply:
done in another place (code has moved dues to previous merge)